feature/bgp/policybase/otg_tests/link_bandwidth_test/README.md

```diff
@@ -0,0 +1,149 @@
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'nomatch_100_set_linkbw_2G' with the following `statements`
+    * statement[name='2-gigabit-match']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'regex_nomatch_as100'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_2G']
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'del_linkbw' with the following `statements`
+    * statement[name='del_linkbw']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'linkbw_any'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_any']
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * For each policy-definition created, run a subtest (RT-7.8.3.x-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify expected communities are present in ATE.
+    * Verify expected communities are present in DUT state.
+      * Do not fail test if this path is not supported, only log results
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+
+    * Expected matches for each policy
+      |              | zero_linkbw                            | match_100_set_linkbw_1M                     |
+      | ------------ | -------------------------------------- | ------------------------------------------- |
+      | prefix-set-1 | [ "link-bandwidth:100:0" ]             | none                                        |
+      | prefix-set-2 | [  "100:100", "link-bandwidth:100:0" ] | [ "100:100", "link-bandwidth:100:1000000" ] |
+      | prefix-set-3 | [ "link-bandwidth:100:0" ]             | [ "link-bandwidth:100:1000000" ]            |
```

Comment:
```suggestion
      | prefix-set-3 | [ "link-bandwidth:100:0" ]             | [ "link-bandwidth:100:0" ]            |
```

Reply:
Done. Also renamed policy to reflect INVERT